main.js

```diff
@@ -387,3 +435,55 @@ export function maxEncodedIdBytes({
   }
   return plainBytes + (multibase ? 1 : 0);
 }
+
+/**
+ * Generates a secret key seed encoded as a string that can be stored and later
+ * used to generate a key pair. The public key from the key pair can be used as
+ * an identifier. The key seed (both raw and encoded form) MUST be kept secret.
+ *
+ * @param {object} [options] - The options to use.
+ * @param {string} [options.encoding='base58'] - Encoding format.
+ * @param {number} [options.bitLength=32 * 8] - Number of bits to generate.
+ * @param {boolean} [options.multibase=true] - Use multibase encoding.
+ * @param {boolean} [options.multihash=true] - Use multihash encoding.
+
+ * @returns {string} - Secret key seed encoded as a string.
+ */
+export async function generateSecretKeySeed({
+  bitLength = 32 * 8,
+  encoding = 'base58',
+  multibase = true,
+  multihash = true
+} = {}) {
+  // reuse `generateId` for convenience, but a key seed is *SECRET* and
+  // not an identifier itself, rather it is used to generate an identifier via
+  // a public key
+  return generateId({bitLength, encoding, multibase, multihash});
```

Comment:
I think we should be explicit about `fixedLength` being `false` even though that's the (current) default. Not using a fixed length of false for a seed is a security problem (it will eliminate some bits of security).

```suggestion
  return generateId({bitLength, encoding, fixedLength: false, multibase, multihash});
```

Reply:
Updated 05cad6f